### DISCUSSION TITLE
Fully sign out an oath user.

### DISCUSSION THREAD

**Discussion by wheelie33** [supabase/supabase#26861] [Questions]
calling await supabaseClient.auth.signOut(); does in fact remove the current Supabase user session but there is no way to trigger an oauth signout.

For instance, I have users that might want to login using a different Azure account and there isn't a way to show the Azure signout page via Supabase.  Any ideas how to achieve this.  (yes, I've googled for the better part of an hour!)

**Comment by wheelie33** [comment]
this may not be the 'right' way but It works.

```
const handleLogout = async (fullLogout: boolean) => {
    await supabaseClient.auth.signOut();
    if (!fullLogout) return;

    const currentUrl = window.location.href;
    const tenantId = import.meta.env.VITE_SUPABASE_KEY;
    const azureLogoutUrl = `https://login.microsoftonline.com/${tenantId}/oauth2/v2.0/logout?post_logout_redirect_uri=${currentUrl}`;
    window.location.href = azureLogoutUrl;
  };
```

**Comment by DominicNetsmartz** [comment]
The solution that worked for me was adding the
`queryParams: {
              access_type: 'offline',
              prompt:'select_accoun',
            } `
             to the options on signin
https://www.reddit.com/r/Supabase/comments/16nkfts/after_logout_when_i_click_on_sign_in_with_google/
